spec/project_spec.rb

```diff
@@ -595,13 +602,16 @@ def callback(_args); end
         method(:callback)
       )
       variation_to_return = project_config.get_variation_from_id('test_experiment', '111128')
+      sleep 0.1
```

Comment:
I am not sure if we need to put a sleep before allow.

Reply:
the thread will not have a chance to run without the sleep.